netket/abstract_driver.py

```diff
@@ -0,0 +1,281 @@
+import abc
+
+from netket._core import deprecated
+import netket as _nk
+
+from netket.logging import JsonLog as _JsonLog
+
+from netket.vmc_common import make_optimizer_fn, tree_map
+
+from tqdm import tqdm
+
+import warnings
+
+
+def obs_stat_to_dict(value):
+    st = value.asdict()
+    st["Mean"] = st["Mean"].real
```

Comment:
I am unsure whether we should discard the imaginary part. In principle, defining a non-hermitian operator in NetKet should work in most places now otherwise, if I'm not mistaken.

Reply:
@gcarleo 

I agree with you that this is weird.
But, again, this is simply the code from the old drivers.
If we change this, this is a breaking change.

Reply:
ok, true.